### DISCUSSION TITLE
datasets

### DISCUSSION THREAD

**Discussion by 1953678487** [open-mmlab/mmdetection#7573] [Q&A]
The number of pictures in my training set is 1440 and the number of samples per GPU is 2. Why is the number of samples behind the epoch printed by training 2160? Is it because of data enhancement? I use the official fasterrcnn for training and convert the data into VOC format.

**Comment by 1953678487** [comment]
my test set num is 360

**Comment by ChenXuuuuu** [comment]
I have same question，Do you know why？

**Comment by jbwang1997** [comment]
The official VOC dataset setting in mmdetection repeats dataset for [3 times](https://github.com/open-mmlab/mmdetection/blob/master/configs/_base_/datasets/voc0712.py#L36)
